### DISCUSSION TITLE
SDK for Embedding dashboards

### DISCUSSION THREAD

**Discussion by kormpakis** [apache/superset#27189] [General]
The documentation is really bad. 
There should be an easy and straightforward way to let us understand what is needed, all in one page.
The docs don't even mention how to acquire an access token, or how to make the charts embeddalby by changing the superset_config.py.

Please update the docs.

**Comment by michael-s-molina** [comment]
Hi @kormpakis. I converted your issue labeled as SIP to a discussion which is more appropriate, given that SIPs include detailed plans on how to resolve a particular problem. 

In regards to your request, are you willing to contribute to improve the documentation? Tagging @sfirke who worked on many doc contributions before.

**Reply by sfirke** [reply]
Thanks for raising this @kormpakis.  Are the docs on embedded Superset bad?  I thought they basically don't exist on the official docs site which is a different and worse problem 😉 

There's this resource from Preset: https://preset.io/events/embedding-your-superset-dashboards-into-your-own-application/ and then people chat in the Slack channel related to embedding Superset.

I agree it would be good to document in our official docs how to use Superset in an embedded context.  I have zero clue about this myself, so can't help write them, but if a community member wishes to contribute this would be a great area to get started!  I can help advise on where in the docs the content should go and help with formatting etc.

**Comment by ltrainstg** [comment]
Some one posted a bit more details in another questions here: 
[Embed superset in react](https://github.com/apache/superset/discussions/26984)

I just got started on looking at open source BI tools today. 
They mention something called: [featrue flags](https://github.com/apache/superset/blob/master/RESOURCES/FEATURE_FLAGS.md?plain=1) in the video so I think I need to change setup.py in the docker initialization steps. I'll look up somewhere how to do that. 
Them the video seems to say that the gui interface gives the data we need and can then use React to host the embedded image?
